File: ml-data.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
from ninstances_py.ninstances import ninstances
from mld_statistics_py.mld_statistics import MdlSta

def main():
    loop_condition=True

    while loop_condition == True:
        print("\nMenu:\n")
        print("0.- EXIT")
        print("1.- Number of instances, features and labels (Juan A. Romero)")
        print("2.- Multi-label Datasets and Statistics (Fernando León")

        main_input = int(input())

        if main_input == 0:
            loop_condition = False
            break
        elif main_input == 1:
            datafile=input("Introduce nombre de datafile: ")
            ninstances(datafile)
        elif main_input == 2:
            datafile = input("Introduce nombre de datafile: ")
            MdlSta(datafile)



if __name__ == "__main__":
    main()

File: ml-data.c
#include <stdio.h>
#include "ninstances-c/ninstances.h"
#include "nlabels-c/nlabels.h"

int main(void)
{
int x, flag;
flag=1;

while(flag==1)
{
printf("Menu:\n");
printf("0.- EXIT\n");
printf("1.- Number of instances, features and labels (Juan A. Romero)\n");
printf("2.- Total of labels of each type. (Juan I. García)\n");
scanf("%d", &x);
fflush(stdin);
printf("\n\n");
switch(x){
    case 0:
        flag=0;
        break;
    case 1:
        printf("Ejecutando la función ninstances() de Juan A. Romero:\n");
        ninstances("yeast.complete");
        break;
    case 2:
        printf("Ejecutando la función nlabels() de Juan I. García:\n");
        nlabels("yeast.complete");
        break;
/* add here more functionality */
    }
}

}
